### ISSUE TITLE
By codelab directions, links are incorrect in icon-toggle-demo.html and icon-toggle.html after bower install

### ISSUE THREAD

**Issue by xeffrei** [googlecodelabs/polymer-first-elements#7]
According to the codelab tutorial the `icon-toggle.html` file should have the following links:

```
<link rel="import"
    href="../bower_components/polymer/polymer.html">
<link rel="import" href="../iron-icon/iron-icon.html">
```

but the directory structure after following the codelab instructions does not agree with this directory structure.

Also, the repository's versions of `icon-toggle.html` and `icon-toggle-demo.html` do not have the correct links at the top of their files.  They ship with

```
<link rel="import" href="../polymer/polymer.html">
<link rel="import" href="../iron-icon/iron-icon.html">
```

and

```
<link rel="import" href="../../polymer/polymer.html">
<link rel="import" href="../../iron-icons/iron-icons.html">
<link rel="import" href="../icon-toggle.html">
```

respectively.

All of these directory structures are at odds with the codelab tutorial since after following the instructions, the directory structure is

```
icon-toggle
    |_ bower-components
    |_ demo
        |_ icon-toggle-demo.html
    |_ bower.json
    |_ icon-toggle.html
```

This seems to be why issue #6 exists. After manually changing the link tags to 

in **demo**:

```
<link rel="import" href="../bower_components/polymer/polymer.html">
<link rel="import" href="../bower_components/iron-icons/iron-icons.html">
<link rel="import" href="../icon-toggle.html">
```

and in **icon-toggle.html**:

```
<link rel="import" href="bower_components/polymer/polymer.html">
<link rel="import" href="bower_components/iron-icon/iron-icon.html">
```

I was able to get the starter text to display on run.

**Comment by przmv** [comment]
Seems it was fixed in #8 — but it hasn't been merged upstream yet.

**Comment by xeffrei** [comment]
Note: This issue comes from following the codelab instructions while using the Chrome dev editor.  #8 explains how the code given is for the build directory so bower-components should not be included in the path. Polyserve should take care of the path for you.
